docs/faq.md

```diff
@@ -3,57 +3,21 @@ title: Frequently Asked Questions
 ---
 
 ### I'm not sure what to do with Reason
-Think of what project you'd usually make if it was pure JavaScript; try porting/writing that in Reason + BuckleScript instead! We recommend trying to make concrete, end-user projects (e.g. a little command line util) rather than infra-level projects (e.g. a boilerplate generator). The latter category requires expertise and understanding idiomatic Reason code.
 
-### What's the relation between Reason, BuckleScript and OCaml?
-Reason's a syntax for OCaml and supports all its features. BuckleScript compiles OCaml/Reason code into JavaScript.
+You can do all the things you'd usually do with OCaml! OCaml is an incredible useful language for systems programming, while still being able to compile to pretty type safe JS with the aid of the `js_of_ocaml` compiler.
 
-### Where do all these `print_endline`, `string_of_int` functions come from?
-They're from the standard library, pre-`open`ed during the compilation of your file. This is why you see them in scope.
+Natively compiled CLI's are also known to be really fast (like... C-like fast), and since the language is garbage collected, you will find yourself in a very nice spot of not having to worry about borrow-checking like in Rust and you don't have to deal with verbose non-ML languages like Go. 
 
-You can read more about the Pervasives library in the api documentation:
+### What is BuckleScript and ReScript, and why is it mentioned in so many Reason related resources?
 
-https://reasonml.github.io/api/Pervasives.html
+Reason used to be tightly coupled to "BuckleScript", which used to be a OCaml-to-JS compiler like JSOO, but with different tweaks to get different JS output.
```

Comment:
Can you replace JSOO with the full js_of_ocaml name? I don't think shortening it helps much, just adds confusion.

I think the portion "but with different tweaks to get different JS output" can just be removed.

Reply:
This part was rewritten (didn't use JSOO after all)